src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/service/placeholders/GenericLetterPlaceholderService.java

```diff
@@ -104,14 +105,14 @@ private String getAppealReference(SscsCaseData caseData) {
 
     private Map<String, Object> getAddressPlaceHolders(Address address) {
         var addressPlaceHolders = new HashMap<String, Object>();
-        String[] lines = lines(address);
-        String[] addressConstants = {LETTER_ADDRESS_LINE_1, LETTER_ADDRESS_LINE_2, LETTER_ADDRESS_LINE_3,
-            LETTER_ADDRESS_LINE_4, LETTER_ADDRESS_POSTCODE};
+        List<String> addressConstants = List.of(LETTER_ADDRESS_LINE_1, LETTER_ADDRESS_LINE_2, LETTER_ADDRESS_LINE_3,
+                LETTER_ADDRESS_LINE_4, LETTER_ADDRESS_POSTCODE);
 
-        for (int i = 0; i < lines.length; i++) {
-            addressPlaceHolders.put(addressConstants[i], truncateAddressLine(defaultToEmptyStringIfNull(lines[i])));
-        }
+        List<String> lines = lines(address);
 
+        for (int i = 0; i < lines.size(); i++) {
```

Comment:
given that this same logic is repeated for most calls to lines(), is it worth incorporating it into the method (or creating a separate method that calls it), and passing a boolean to decide whether or not to truncate